tests/test_default_qubit.py

```diff
@@ -681,15 +714,15 @@ def test_sample_values(self, qubit_device_2_wires, tol):
 class TestMaps:
     """Tests the maps for operations and observables."""
 
-    def test_operation_map(self, qubit_device_2_wires):
+    def test_operation_map(self, qubit_device_3_wires):
         """Test that default qubit device supports all PennyLane discrete gates."""
         
-        assert set(qml.ops._qubit__ops__) ==  set(qubit_device_2_wires._operation_map)
+        assert set(qml.ops._qubit__ops__) ==  set(qubit_device_3_wires._operation_map)
 
-    def test_observable_map(self, qubit_device_2_wires):
+    def test_observable_map(self, qubit_device_3_wires):
         """Test that default qubit device supports all PennyLane discrete observables."""
         
-        assert set(qml.ops._qubit__obs__) | {"Identity"} == set(qubit_device_2_wires._observable_map)
+        assert set(qml.ops._qubit__obs__) | {"Identity"} == set(qubit_device_3_wires._observable_map)
```

Comment:
Todo for all of the above: Unrelated tests should not have to be updated like this when new code is added. This test should be modified to make it independent on the number of wires in the initialized device.